Formula/terraform-provider-lxd.rb

```diff
@@ -0,0 +1,17 @@
+class TerraformProviderLxd < Formula
+  desc "LXD Resource provider for Terraform"
+  homepage "https://github.com/terraform-lxd/terraform-provider-lxd"
```

Comment:
what is the use case of installing a terraform plugin?

Reply:
Terraform allows for provisioning of resources on various platforms. This package specifically allows for terraform to connect to and provision containers and virtual machines on an LXD server or cluster of servers.

Reply:
actually, I thought this was more like the libvirt plugin but now not so much.

Reply:
Yeah, I saw the libvirt plugin as well, not sure how folks are using that either.

I dont know what we're trying to achieve things outside download the provider via terraform (doesn't `terraform init` would solve the provider installation issue?)

Reply:
It has for this plugin, I have not read up on all the requirements for the terraform registry. As in it should be able to point you to a binary that works for your platform. Libvirt requires a system library to build/link against.
In this case it is pure go.. so no actual need to have it in hombrew, I just got carried away while packaging other stuff :)